frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -143,6 +146,74 @@ export default {
     },
     symbol () {
       return currencies[this.groupSettings.mincomeCurrency].symbol
+    },
+    paymentsReceived () {
+      return 'TODO'
+    },
+    paymentsTodo () {
+      const monthlyPayments = this.groupMonthlyPayments
+      const cMonthstamp = currentMonthstamp()
+      const pMonthstamp = prevMonthstamp(cMonthstamp)
+      const paymentsTodo = []
+      const pastMonth = monthlyPayments[pMonthstamp]
+      // const currentDistribution = this.groupIncomeAdjustedDistribution
+      const currentDistribution = this.paymentsDistribution
+      const pledgeAmount = this.ourGroupProfile['pledgeAmount']
+      // first we need to check if there are any late payments that need to be added to the paymentsTodo list
+      if (pastMonth) {
+        const previousDistribution = pastMonth.lastAdjustedDistribution
+        // this creates a date at the end of the previous month
+        // (see comment in time.js:dateFromMonthstamp())
+        // const date = new Date(monthstamp) // TODO: return this in a computed property called prevMonthDueDate and reference that in the pug instead of repeatedly adding it to this data
+        for (const payment of previousDistribution) {
+          if (payment.from === this.ourUsername && payment.amount > 0) {
+            // Let A = the amount we owe from the previous distribution.
+            // Let B = the total we've sent to payment.to from the current
+            //         month's paymentsFrom.
+            // Let C = the total amount we "owe" to payment.to from the
+            //         current month's distribution.
+            // Let D = the amount we're pledging this month
+            // Let E = the amount still unpaid for the previous month's distribution,
+            //         calculated as: C > 0 ? A : A + D - B
```

Comment:
I understand your reason to "minify" the variable names. What if we take a middle step, where we can reduce the nr of characters while still understanding the meaning of the variables:
```
A -> prevOwe
B -> sent
C -> curOwe
D -> pldg
C -> unpaid
```

Clarity is important, especially when dealing with complex logic.

Reply:
Is it OK if we keep it as-is? If it really bugs you feel free to modify it in a future PR and adjust the spacing in the comments so that everything lines up pretty!